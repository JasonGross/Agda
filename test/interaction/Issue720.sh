BASE=Issue720
LOAD="IOTCM \"$BASE.agda\" Interactive Direct (Cmd_load \"$BASE.agda\" [])\n"

rm -f $BASE.pipe
mkfifo $BASE.pipe

agda --interaction < $BASE.pipe 2>&1 &

rm -f $BASE.agdai
( printf "$LOAD"; \
  while [ ! -e $BASE.agdai ]; do \
    sleep 0.1; \
  done; \
  sleep 2; \  # On some systems time stamps may have a resolution of 1s.
  touch $BASE.agda; \
  printf "$LOAD" \
) > $BASE.pipe

rm -f $BASE.pipe
