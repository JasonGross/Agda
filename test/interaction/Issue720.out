Agda2> (agda2-status-action "")
(agda2-info-action "*Type-checking*" "" nil)
(agda2-highlight-clear)
(agda2-highlight-add-annotations '(1 36 (comment) nil) '(38 44 (keyword) nil) '(45 54 (module) nil ("agda-default-include-path/Agda/Prim.agda" . 1)) '(55 60 (keyword) nil) '(62 134 (comment) nil) '(135 153 (comment) nil) '(154 226 (comment) nil) '(228 234 (keyword) nil) '(235 236 (number) nil) '(242 251 (keyword) nil) '(254 259 (postulate) nil ("agda-default-include-path/Agda/Prim.agda" . 254)) '(260 261 (symbol) nil) '(262 265 (primitivetype) nil) '(268 273 (postulate) nil ("agda-default-include-path/Agda/Prim.agda" . 268)) '(274 275 (symbol) nil) '(276 281 (postulate) nil ("agda-default-include-path/Agda/Prim.agda" . 254)) '(284 288 (postulate) nil ("agda-default-include-path/Agda/Prim.agda" . 284)) '(290 291 (symbol) nil) '(292 293 (symbol) nil) '(293 294 (bound) nil ("agda-default-include-path/Agda/Prim.agda" . 293)) '(295 296 (symbol) nil) '(297 302 (postulate) nil ("agda-default-include-path/Agda/Prim.agda" . 254)) '(302 303 (symbol) nil) '(304 305 (symbol) nil) '(306 311 (postulate) nil ("agda-default-include-path/Agda/Prim.agda" . 254)) '(314 317 (postulate operator) nil ("agda-default-include-path/Agda/Prim.agda" . 314)) '(320 321 (symbol) nil) '(322 323 (symbol) nil) '(323 325 (bound) nil ("agda-default-include-path/Agda/Prim.agda" . 323)) '(326 328 (bound) nil ("agda-default-include-path/Agda/Prim.agda" . 326)) '(329 330 (symbol) nil) '(331 336 (postulate) nil ("agda-default-include-path/Agda/Prim.agda" . 254)) '(336 337 (symbol) nil) '(338 339 (symbol) nil) '(340 345 (postulate) nil ("agda-default-include-path/Agda/Prim.agda" . 254)) '(347 414 (comment) nil) '(415 458 (comment) nil) '(460 463 (symbol) nil) '(464 477 (keyword) nil) '(478 483 (postulate) nil ("agda-default-include-path/Agda/Prim.agda" . 254)) '(492 495 (symbol) nil) '(496 499 (symbol) nil) '(500 508 (keyword) nil) '(509 514 (postulate) nil ("agda-default-include-path/Agda/Prim.agda" . 268)) '(528 531 (symbol) nil) '(532 535 (symbol) nil) '(536 544 (keyword) nil) '(545 549 (postulate) nil ("agda-default-include-path/Agda/Prim.agda" . 284)) '(564 567 (symbol) nil) '(568 571 (symbol) nil) '(572 580 (keyword) nil) '(581 584 (postulate operator) nil ("agda-default-include-path/Agda/Prim.agda" . 314)) '(600 603 (symbol) nil) '(605 608 (symbol) nil) '(609 616 (keyword) nil) '(627 632 (postulate) nil ("agda-default-include-path/Agda/Prim.agda" . 254)) '(634 637 (symbol) nil) '(638 641 (symbol) nil) '(642 649 (keyword) nil) '(660 665 (primitive) nil ("agda-default-include-path/Agda/Prim.agda" . 268)) '(667 670 (symbol) nil) '(671 674 (symbol) nil) '(675 682 (keyword) nil) '(693 697 (primitive) nil ("agda-default-include-path/Agda/Prim.agda" . 284)) '(700 703 (symbol) nil) '(704 707 (symbol) nil) '(708 715 (keyword) nil) '(726 729 (primitive operator) nil ("agda-default-include-path/Agda/Prim.agda" . 314)) '(733 736 (symbol) nil))
(agda2-info-action "*Type-checking*" "Checking Issue720 (Issue720.agda).\n" t)
(agda2-highlight-add-annotations '(1 7 (keyword) nil) '(17 22 (keyword) nil))
(agda2-highlight-add-annotations '(1 7 (keyword) nil) '(8 16 (module) nil ("Issue720.agda" . 1)) '(17 22 (keyword) nil))
(agda2-highlight-add-annotations '(1 7 (keyword) nil) '(8 16 (module) nil ("Issue720.agda" . 1)) '(17 22 (keyword) nil))
(agda2-info-action "*Type-checking*" "Finished Issue720.\n" t)
(agda2-status-action "Checked")
(agda2-info-action "*All Goals*" "" nil)
((last . 1) . (agda2-goals-action '()))
Agda2> (agda2-status-action "")
(agda2-info-action "*Type-checking*" "" nil)
(agda2-highlight-clear)
(agda2-highlight-add-annotations '(1 36 (comment) nil) '(38 44 (keyword) nil) '(45 54 (module) nil ("agda-default-include-path/Agda/Prim.agda" . 1)) '(55 60 (keyword) nil) '(62 134 (comment) nil) '(135 153 (comment) nil) '(154 226 (comment) nil) '(228 234 (keyword) nil) '(235 236 (number) nil) '(242 251 (keyword) nil) '(254 259 (postulate) nil ("agda-default-include-path/Agda/Prim.agda" . 254)) '(260 261 (symbol) nil) '(262 265 (primitivetype) nil) '(268 273 (postulate) nil ("agda-default-include-path/Agda/Prim.agda" . 268)) '(274 275 (symbol) nil) '(276 281 (postulate) nil ("agda-default-include-path/Agda/Prim.agda" . 254)) '(284 288 (postulate) nil ("agda-default-include-path/Agda/Prim.agda" . 284)) '(290 291 (symbol) nil) '(292 293 (symbol) nil) '(293 294 (bound) nil ("agda-default-include-path/Agda/Prim.agda" . 293)) '(295 296 (symbol) nil) '(297 302 (postulate) nil ("agda-default-include-path/Agda/Prim.agda" . 254)) '(302 303 (symbol) nil) '(304 305 (symbol) nil) '(306 311 (postulate) nil ("agda-default-include-path/Agda/Prim.agda" . 254)) '(314 317 (postulate operator) nil ("agda-default-include-path/Agda/Prim.agda" . 314)) '(320 321 (symbol) nil) '(322 323 (symbol) nil) '(323 325 (bound) nil ("agda-default-include-path/Agda/Prim.agda" . 323)) '(326 328 (bound) nil ("agda-default-include-path/Agda/Prim.agda" . 326)) '(329 330 (symbol) nil) '(331 336 (postulate) nil ("agda-default-include-path/Agda/Prim.agda" . 254)) '(336 337 (symbol) nil) '(338 339 (symbol) nil) '(340 345 (postulate) nil ("agda-default-include-path/Agda/Prim.agda" . 254)) '(347 414 (comment) nil) '(415 458 (comment) nil) '(460 463 (symbol) nil) '(464 477 (keyword) nil) '(478 483 (postulate) nil ("agda-default-include-path/Agda/Prim.agda" . 254)) '(492 495 (symbol) nil) '(496 499 (symbol) nil) '(500 508 (keyword) nil) '(509 514 (postulate) nil ("agda-default-include-path/Agda/Prim.agda" . 268)) '(528 531 (symbol) nil) '(532 535 (symbol) nil) '(536 544 (keyword) nil) '(545 549 (postulate) nil ("agda-default-include-path/Agda/Prim.agda" . 284)) '(564 567 (symbol) nil) '(568 571 (symbol) nil) '(572 580 (keyword) nil) '(581 584 (postulate operator) nil ("agda-default-include-path/Agda/Prim.agda" . 314)) '(600 603 (symbol) nil) '(605 608 (symbol) nil) '(609 616 (keyword) nil) '(627 632 (postulate) nil ("agda-default-include-path/Agda/Prim.agda" . 254)) '(634 637 (symbol) nil) '(638 641 (symbol) nil) '(642 649 (keyword) nil) '(660 665 (primitive) nil ("agda-default-include-path/Agda/Prim.agda" . 268)) '(667 670 (symbol) nil) '(671 674 (symbol) nil) '(675 682 (keyword) nil) '(693 697 (primitive) nil ("agda-default-include-path/Agda/Prim.agda" . 284)) '(700 703 (symbol) nil) '(704 707 (symbol) nil) '(708 715 (keyword) nil) '(726 729 (primitive operator) nil ("agda-default-include-path/Agda/Prim.agda" . 314)) '(733 736 (symbol) nil))
(agda2-info-action "*Type-checking*" "Checking Issue720 (Issue720.agda).\n" t)
(agda2-highlight-add-annotations '(1 7 (keyword) nil) '(17 22 (keyword) nil))
(agda2-highlight-add-annotations '(1 7 (keyword) nil) '(8 16 (module) nil ("Issue720.agda" . 1)) '(17 22 (keyword) nil))
(agda2-highlight-add-annotations '(1 7 (keyword) nil) '(8 16 (module) nil ("Issue720.agda" . 1)) '(17 22 (keyword) nil))
(agda2-info-action "*Type-checking*" "Finished Issue720.\n" t)
(agda2-status-action "Checked")
(agda2-info-action "*All Goals*" "" nil)
((last . 1) . (agda2-goals-action '()))
Agda2> 